recipes/pyutilib/meta.yaml

```diff
@@ -97,7 +97,3 @@ about:
   license: BSD License
   summary: 'PyUtilib: A collection of Python utilities'
 
-extras:
-  recipe-maintainers:
-    - whart222
```

Comment:
You have to add this back because the feedstock will be generated using this info.